docker/docker-compose.yml

```diff
@@ -39,29 +41,28 @@ services:
     deploy: *default-deploy
     volumes:
       - grafana-storage:/var/lib/grafana
-    environment:
-      - GF_INSTALL_PLUGINS=grafana-clock-panel,grafana-influx-flux-datasource
```

Comment:
These aren't needed?

Reply:
No, it actually prevents the container from coming up if present.